brands/shop/newagent.json

```diff
@@ -0,0 +1,32 @@
+{
+  "shop/newagent|Martin's": {
+    "countryCodes": ["gb"],
+    "match": [
+      "shop/convenience|Martin's",
+      "shop/newagent|McColls"
+    ],
+    "nocount": true,
+    "tags": {
+      "brand": "Martin's",
+      "brand:wikidata": "Q7277785",
+      "brand:wikipedia": "en:RS McColl",
+      "name": "Martin's",
+      "shop": "convenience"
+    }
+  },
+  "shop/newagent|McColls": {
+    "countryCodes": ["gb"],
+    "match": [
+      "shop/convenience|Martin's",
+      "shop/newsagent|Martin's"
+    ],
+    "nocount": true,
+    "tags": {
+      "brand": "Martin's",
+      "brand:wikidata": "Q7277785",
+      "brand:wikipedia": "en:RS McColl",
+      "name": "Martin's",
+      "shop": "newsagent"
+    }
+  }
+}
```

Comment:
This file seems wrong to me. It is not called `newagent` but rather `newsagent`...

Reply:
Can you specify where you saw 'newagent'?